### PR TITLE
Migration cleanup

### PR DESCRIPTION
**Pull request by matzew**
Some clean ups for migration, also removed the old `.sql` scripts as well as adding an initial `README` for instructions.

### CODE REVIEW

**Comment by matzew** [comment]
@qmx @sebastienblanc @danielpassos please review the changes

File: migrator/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# UnifiedPush Server - Migration guide
+
+For the database migration you may need to run a simple shell script
```
Comment (sebastienblanc) [inline migrator/README.md]: let's add a final period `.`

File: migrator/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+# 1.0.x users
+
+For the migration to our 1.0.3 release we are only supporting the 1.0.x series of the UnifiedPush Server:
+
+* 1.0.0
+* 1.0.1
+* 1.0.2
+
+In terms of supported databases, we are supporting MySQL 5.5 and PostgreSQL 9.
```
Comment (sebastienblanc) [inline migrator/README.md]: Should we mention MariaDB is also part of "MySQL" ?

Comment (matzew) [inline migrator/README.md]: well, not sure... on explicit MariaDB mentioning here... Opebshift still
labels its DB as MySQL

On Wed, Mar 11, 2015 at 10:54 AM, Sebastien Blanc <notifications@github.com>
wrote:

> In migrator/README.md
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/510#discussion_r26199568>
> :
>
> > @@ -0,0 +1,63 @@
> > +# UnifiedPush Server - Migration guide
> > +
> > +For the database migration you may need to run a simple shell script
> > +
> > +# 1.0.x users
> > +
> > +For the migration to our 1.0.3 release we are only supporting the 1.0.x series of the UnifiedPush Server:
> > +
> > +* 1.0.0
> > +* 1.0.1
> > +* 1.0.2
> > +
> > +In terms of supported databases, we are supporting MySQL 5.5 and PostgreSQL 9.
>
> Should we mention MariaDB is also part of "MySQL" ?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aerogear/aerogear-unifiedpush-server/pull/510/files#r26199568>
> .
>



-- 
Matthias Wessendorf

blog: http://matthiaswessendorf.wordpress.com/
sessions: http://www.slideshare.net/mwessendorf
twitter: http://twitter.com/mwessendorf

File: migrator/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+Below you find information on how to perform the migration on each of the supported databases.
+
+#### PostgreSQL
+
+Inside of the extracted zip file, there is a `liquibase-postgresql-example.properties`, copy that to `liquibase.properties`.
+
+    cp liquibase-postgresql-example.properties liquibase.properties
+
+Once done, you need to edit the new file to match your database credentials.
```
Comment (sebastienblanc) [inline migrator/README.md]: we could add "and database name" , I had to change that since it was postfix with some unwanted extra chars

**Comment by sebastienblanc** [comment]
:+1:

**Pull request merged**